### ISSUE TITLE
CloudSQL unbind no longer working

### ISSUE THREAD

**Issue by marcohelmerich** [GoogleCloudPlatform/gcp-service-broker#463]
We are using CloudSQL MySQL for several month now and are experiencing this issue since about two weeks.
When we try to unbind a mysql service we see this error:
HTTPError 400: Missing parameter: host.

I can reproduce this error with the same user/database via gcloud command. In the source code, while creating an SQL User GCP Service broker hardcodes the user to ''. I couldn't get gcloud cli to work with a '' host. I can only delete Users with a host set (% for everyhost)

Did the CloudSQL API Change and the host needs to be adjusted?

**Comment by marcohelmerich** [comment]
I opened a case with google support and they told me that this is already a known issue
gcloud sql users delete test --instance=pcf-sb-1-1558611745454979620 --host=' ' # please note the host is a space, not an empty string

"The Cloud SQL specialist team is already aware of this issue and exchanged some thoughts about it. However, there is no meaningful update since mid-May."

I think it would be enough to just replace '' to ' ' here:

https://github.com/GoogleCloudPlatform/gcp-service-broker/blob/378ccb5dd3a26a4b0ac6c7225df67a8f64791f6a/pkg/providers/builtin/cloudsql/sql_account_manager.go#L99

**Comment by josephlewis42** [comment]
Hi @marcohelmerich,

Thanks for the feedback, I'm taking a look at this now. I have your proposed fix in and I'm running integration tests. After validating it I'll batch it up with two other pending changes and start the release process--it's quick to get changes to this repository, but PivNet may take a bit longer if you're waiting on an official tile. 

Thanks!
\- Joseph

**Comment by marcohelmerich** [comment]
Hi @josephlewis42 
thanks. We found your Concourse Pipelines :-) and were able to build the broker and tile ourselves with the fix. Working fine at the moment. We try to set up some CI/CD Pipeline for this so that we can fix future bugs more easily. Actually we have found another bug last friday concerning High Available CloudSQL MySQL INstances. We will add a new Issue (and hopefully a PR) this week too.
